lib/giphify_api/search/query.ex

```diff
@@ -0,0 +1,29 @@
+defmodule GiphifyApi.Query do
+  import Ecto.Query
+  import Ecto.Changeset
+
+  alias GiphifyApi.Repo
+  alias GiphifyApi.Search
+  alias GiphifyApi.Giphy
+
+  def create_search(attrs \\ %{}) do
+    %Search{}
+    |> Search.changeset(attrs)
+    |> Repo.insert()
+  end
+
+  def store_gif_url() do
```

Comment:
I'm really unsure how to test this method. I believe that if Repo.update() will return an `{:ok, struct}` if everything is successful, and a `{:error, changeset}` if the update fails. Should my test asset that an {:ok, struct} is returned when valid data is provided, and {:error, changeset} otherwise?

The docs for Repo.update!(): https://hexdocs.pm/ecto/Ecto.Repo.html#c:update/2